erpnext/hr/doctype/employee/employee.json

```diff
@@ -602,30 +602,30 @@
   {
    "fieldname": "marital_status",
    "fieldtype": "Select",
+   "hidden": 1,
    "label": "Marital Status",
-   "options": "\nSingle\nMarried\nDivorced\nWidowed",
-   "hidden": 1
+   "options": "\nSingle\nMarried\nDivorced\nWidowed"
   },
   {
    "fieldname": "blood_group",
    "fieldtype": "Select",
+   "hidden": 1,
    "label": "Blood Group",
-   "options": "\nA+\nA-\nB+\nB-\nAB+\nAB-\nO+\nO-",
-   "hidden": 1
+   "options": "\nA+\nA-\nB+\nB-\nAB+\nAB-\nO+\nO-"
   },
   {
    "description": "Here you can maintain family details like name and occupation of parent, spouse and children",
    "fieldname": "family_background",
    "fieldtype": "Small Text",
-   "label": "Family Background",
-   "hidden": 1
+   "hidden": 1,
```

Comment:
- Can we unhide all the fields ?

Reply:
I have just changed the settings in last line rest is default changes .

Reply:
@hrwX let's not hhange the old configurations, it was hidden earlier as well.